.github/workflows/release.yaml

```diff
@@ -15,4 +15,10 @@ jobs:
         uses: marvinpinto/action-automatic-releases@919008cf3f741b179569b7a6fb4d8860689ab7f0 # v1.2.1
         with:
           repo_token: "${{ secrets.GITHUB_TOKEN }}"
-          prerelease: false
\ No newline at end of file
+          prerelease: false
+      - name: Install bom and test presence in path
```

Comment:
If we aren't shipping a binary and are just shipping a Go library, do we still need SBOM generation? I'm less familiar with the details here. I know that it's good to ship one with all images builds that you might have, but less clear on package libraries